triedb/pathdb/database.go

```diff
@@ -362,6 +430,7 @@ func (db *Database) Enable(root common.Hash) error {
 	// reset the persistent state id back to zero.
 	batch := db.diskdb.NewBatch()
 	rawdb.DeleteTrieJournal(batch)
+	rawdb.DeleteSnapshotRoot(batch)
```

Comment:
Could you please explain why we need to delete the snapshot root here, I don't really understand